terraform/README.md

```diff
@@ -30,29 +43,33 @@ TODO
 
 Add the following to your main.tf for the canonical k8s solution:
 
-```
+```hcl
 module "k8s" {
   source        = "git::https://github.com/canonical/k8s-bundles//terraform?ref=main" 
-  model         = "my-canonical-k8s"
+  model         = {
```

Comment:
I think its worth leaving a note here that you can import an existing model 

something close to `terraform import module.k8s.juju_model.this <model_name>`

the reference may be slightly off there.